README.md

```diff
@@ -2,3 +2,36 @@
 
 The Empirical Software Engineering journal (EMSE) wants to encourage open-science and reproducible research.
 This repository describes the process and infrastructure that support this initiative.
+
+## Open-science Board
+
+Chairs: TBA
+
+Board members: TBA
+
+## Process
+
+To give about this proposal, there are two possibilities:
+* comment on the pull-request discussion tread
+* comment a specific line by clicking in the left margin)
+
+Process description:
+
+1. Once a paper is accepted, the authors have the choice to submit a replication package
```

Comment:
Could we somehow prep people for this already in the feedback email they get upon a Minor revision? (Or also for Major revisions since some papers might go directly from MAjor to accept) Something like this text in the revision email template: 
"EMSE encourages open science and reproducible research. Not only is this good for science and the community as a whole (since progress can be faster), there is also strong evidence (Ref/Link?) that it leads to more citations and more impact/interest. A revision is no guarantee of future acceptance but if your paper is eventually accepted for publication in EMSE you will have the option to provide a replication package. Please consider and prepare for this already now. To get more information about the open-science initiative, please visit https://github.com/emsejournal/openscience"

Reply:
Agree. Done in new version.